src/functions/validateLevelFunc.tsx

```diff
@@ -0,0 +1,29 @@
+export function validateLevel(guessedWords: number, updateLevelFunc: (level: number) => void) {
+  if (guessedWords > 1) {
```

Comment:
rework this logic

Reply:
Need to discuss this with you in real-time, but will switch work here instead of ifs? Or maybe smth like useReducer hook?

Reply:
Okay, I think I found better implementation) Smth like this: should work?

  guessedWords < 3 ? updateLevelFunc(1) : 
  guessedWords < 6 ? updateLevelFunc(2) :
  guessedWords < 10 ? updateLevelFunc(3) :
  guessedWords < 14 ? updateLevelFunc(4) :
  guessedWords < 18 ? updateLevelFunc(5) :
  guessedWords < 22 ? updateLevelFunc(6) :
  guessedWords < 26 ? updateLevelFunc(7) :
  guessedWords < 30 ? updateLevelFunc(8) :
  guessedWords < 34 ? updateLevelFunc(9) :
  guessedWords > 33 && updateLevelFunc(10)